### ISSUE TITLE
Duplicates between EarthConstants and PlanetParameters 

### ISSUE THREAD

**Issue by trontrytel** [climate-machine/CLIMA#818]
@charleskawczynski - I have noticed that some parameters are defined twice in EarthConstants and PlanetParameters. Some have the same name (like `grav`, `mollmass_water` or `LH_v0`) and are just defined twice. Others have different names (like `rho_liq` and `rho_cloud_liq`) but describe the same thing.
Is that a bug or a feature right now? 

# For CLIMA Developers

- [ ] This feature can be added (if it cannot be, explain why in a comment below -- lack of technical expertise, not relevant to the scope of this project, too ambitious)
- [ ] There is a timeline for when this feature can be implemented
- [ ] The feature has been (or will be) implemented (Please link the PR)

**Comment by charleskawczynski** [comment]
Yes, this was actually done intentionally. We had to both permit default values for moist thermodynamics and throw errors by default in the source code if a `ParameterSet` was not specified. I brought this up with Simon today and, at first glance, and we didn't see a clear way to safely define a single set of parameters, throw errors if they're not defined, and allow for defaults in moist thermo. Maybe it would be clearer if we `EarthParameters` to `MoistThermoDefaults`?

Long story short, we have to think more about this.

**Comment by trontrytel** [comment]
ok, thanks. Yes maybe having file names like ...Parameters and ...DefaultParameters would help. 

Then the parameter names should be the same in the two files? i.e `rho_liq` and `rho_cloud_liq` should be one name?

**Comment by tapios** [comment]
The `EarthParameters` are not just for `MoistThermodynamics` but will be used globally (for soil, sea ice etc.)

**Comment by charleskawczynski** [comment]
Closed by #962
